cpp/dolfinx/graph/KaHIP.cpp

```diff
@@ -20,9 +20,8 @@ using namespace dolfinx;
 
 graph::AdjacencyList<std::int32_t> dolfinx::graph::KaHIP::partition(
     MPI_Comm mpi_comm, int nparts,
-    const graph::AdjacencyList<unsigned long long>& adj_graph)
+    const graph::AdjacencyList<unsigned long long>& adj_graph, bool ghosting)
```

Comment:
Could we used fixed-width ints in place of `unsigned long long`?

Reply:
It is written into KaHIP. Not sure what are the implications for maintaining compatibility on all systems.